src/index.html

```diff
@@ -1,22 +1,23 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta
       http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
+      content="ie=edge">
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
-    />
+      href="./styles/index.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```

Comment:
probably u've forgotten about this part
![image](https://github.com/mate-academy/layout_stop-watch/assets/55200530/f4128dc4-357f-4cd1-82e8-b26dd0293b6b)